### PR TITLE
Chapter2/Homework/3

### PR DESCRIPTION
**Pull request by jeanroyce**

### CODE REVIEW

File: Src/BootCamp.Chapter/MsFactory.cs [hunk 1]
```diff
@@ -1,15 +1,71 @@
...
 
 namespace BootCamp.Chapter
 {
-    public class MsFactory
+    public class MsFactory : Factory
     {
         public DesktopComputer Assemble()
         {
-            return new DesktopComputer();
+            var pc = new DesktopComputer();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/MsFactory.cs:12]: If you compare computer assembling in Ms and Mac, what's the difference? The assemble is exact same, except that individual steps are different.

Comment (Almantask) [reply Src/BootCamp.Chapter/MsFactory.cs:12]: This flow (assembling) can be implemented in abstract class.

File: Src/BootCamp.Chapter/Parts.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Parts
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Parts.cs:3]: Please remove redundant class.

**Comment by jeanroyce** [comment]
Don't know if is correct, but I think abstract class make more sense in my mind now! Thanks :D

**Comment by jeanroyce** [comment]
Maybe this time is correct, maybe 😅

**Pull request closed without merging**